packages/nexrender-provider-gs/src/index.js

```diff
@@ -63,6 +63,12 @@ const upload = (job, settings, src, params) => {
                 cacheControl: params.cacheControl
             }
         }
+        if (params.resumable) {
+            options = {
```

Comment:
@francis-devopso @inlife 

If we pass `params.resumable: false`, it will never get inside this if-statement. Also, `options` is a const and can't be re-assigned.

```js
if (params.resumable !== undefined) {
    options.resumable = params.resumable
}
```

Also, `contentType` above is set on `options.metadata` object, but it might be set on `options` object:

```ts
export interface CreateWriteStreamOptions extends CreateResumableUploadOptions {
    contentType?: string;
    gzip?: string | boolean;
    resumable?: boolean;
    validation?: string | boolean;
}
```